pkg/snapstore/swift_snapstore.go

```diff
@@ -52,8 +56,14 @@ const (
 )
 
 // NewSwiftSnapStore create new SwiftSnapStore from shared configuration with specified bucket
-func NewSwiftSnapStore(bucket, prefix, tempDir string, maxParallelChunkUploads uint) (*SwiftSnapStore, error) {
-	authOpts, err := clientconfig.AuthOptions(nil)
+func NewSwiftSnapStore(config *brtypes.SnapstoreConfig) (*SwiftSnapStore, error) {
+	var clientOpts *clientconfig.ClientOpts
+	if config.IsSource {
+		clientOpts = &clientconfig.ClientOpts{
+			EnvPrefix: envPrefixSource,
+		}
+	}
```

Comment:
Also here I would extract this in a helper function, even though it's used only once, e.g. `getClientOpts(config.IsSource)`